src/bundles/c/stdlib/assert.c

```diff
@@ -361,9 +361,9 @@ const uint8_t tjs__assert[8835] = {
  0x75, 0x72, 0x6e, 0x20, 0x61, 0x20, 0x21, 0x3d,
```

Comment:
(random comment placement).

What did you use to regenerate the code? You only changed a polyfill, so stdlib shouldn't be regenerated I think.

Reply:
`make js`. Yes, you're right. I think I was on an unclean checkout (on another notebook) when force pushing it last night.